package.json

```diff
@@ -51,6 +51,7 @@
     "nsp": "^2.6.3",
     "pify": "^3.0.0",
     "sinon": "^2.3.8",
-    "standard": "^9.0.1"
+    "standard": "^9.0.1",
+    "await-sleep": "^0.0.1"
```

Comment:
Use `delay` instead, it's high quality and widely used.

Reply:
Updated.